Include/cpython/code.h

```diff
@@ -117,12 +118,12 @@ PyAPI_DATA(PyTypeObject) PyCode_Type;
 PyAPI_FUNC(PyCodeObject *) PyCode_New(
         int, int, int, int, int, PyObject *, PyObject *,
         PyObject *, PyObject *, PyObject *, PyObject *,
-        PyObject *, PyObject *, int, PyObject *);
+        PyObject *, PyObject *, int, PyObject *, PyObject *);
```

Comment:
I have doubts about changing the C API. See, we have `PyCode_New` and `PyCode_NewWithPosOnlyArgs` because of new parameter. You need to introduce a new constructor function, or call the old function and change the code object after this with a new setter function.

Reply:
Totally agree with you, it will be better to be backward compatible with previous versions.
I have added function `PyCode_NewWithAnnotations` in order to create `CodeObject` with annotations.